src/frontend/screens/Settings/ProjectSettings/RemoteArchive/RemoteArchiveOn.tsx

```diff
@@ -0,0 +1,237 @@
+  },
+  tracks: {
+    id: 'ProjectSettings.RemoteArchive.tracks',
+    defaultMessage: 'Tracks',
+  },
+  deviceNames: {
+    id: 'ProjectSettings.RemoteArchive.deviceNames',
+    defaultMessage: 'Device Names',
+  },
+  projectSettings: {
+    id: 'ProjectSettings.RemoteArchive.projectSettings',
+    defaultMessage: 'Project Settings (categories, questions)',
+  },
+  serverName: {
+    id: 'ProjectSettings.RemoteArchive.serverName',
+    defaultMessage: 'Server Name',
+  },
+  dateAdded: {
+    id: 'ProjectSettings.RemoteArchive.dateAdded',
+    defaultMessage: 'Date Added',
+  },
+  //   remove: {
+  //     id: 'ProjectSettings.RemoteArchive.remove',
+  //     defaultMessage: 'Remove Server',
+  //   },
+});
+
+export const RemoteArchiveOn: NativeNavigationComponent<'RemoteArchiveOn'> = ({
+  navigation,
+}) => {
+  const {formatMessage} = useIntl();
+  const {data: role, isPending: roleIsPending} = useGetOwnRole();
+  const {data: remoteArchive, isPending} = useGetRemoteArchives();
+
+  const isCoordinator =
+    role?.roleId === COORDINATOR_ROLE_ID || role?.roleId === CREATOR_ROLE_ID;
+
+  const currentRemoteArchive = !remoteArchive ? undefined : remoteArchive[0];
+
+  const handleGoBack = React.useCallback(() => {
+    navigation.navigate('ProjectSettings');
+  }, [navigation]);
+
+  React.useLayoutEffect(() => {
+    navigation.setOptions({
+      headerLeft: props => (
+        <CustomHeaderLeft
+          onPress={handleGoBack}
+          headerBackButtonProps={props}
+        />
+      ),
+    });
+  }, [handleGoBack, navigation]);
+
+  useFocusEffect(
+    React.useCallback(() => {
+      const onBackPress = () => {
+        handleGoBack();
+        return true;
+      };
+
+      const subscription = BackHandler.addEventListener(
+        'hardwareBackPress',
+        onBackPress,
+      );
+
+      return () => subscription.remove();
+    }, [handleGoBack]),
+  );
+
+  if (isPending || roleIsPending) {
+    return <Loading />;
+  }
+
+  return (
+    <ScrollView
+      style={styles.container}
+      contentContainerStyle={{paddingBottom: 60}}>
+      <HeaderText variant="header2" style={styles.title}>
+        {formatMessage(m.remoteArchiveOn)}
+      </HeaderText>
+      <BodyText style={{marginTop: 20}}>
+        {formatMessage(m.syncWithInternet)}
+      </BodyText>
+
+      {isCoordinator ? (
+        <>
+          {/* <Text>{formatMessage(m.remove)}</Text> */}
+          <BodyText variant="smallMeta" style={{marginTop: 20}}>
+            {formatMessage(m.thisIncludes)}
+          </BodyText>
+          <BodyText variant="smallMeta">
+            {formatMessage(m.observations)}
+          </BodyText>
+          <BodyText variant="smallMeta">{formatMessage(m.tracks)}</BodyText>
+          <BodyText variant="smallMeta">
+            {formatMessage(m.deviceNames)}
+          </BodyText>
+          <BodyText variant="smallMeta">
+            {formatMessage(m.projectSettings)}
+          </BodyText>
```

Comment:
There is a slight discrepency here between figma. Figma does not use any of the type ramp, so I have asked for clarification but in the mean time i think it is ok to ignore